src/js/components/Booking.js

```diff
@@ -39,11 +39,11 @@ class Booking {
     };
 
     const urls = {
-      booking: settings.db.url + '/' + settings.db.booking
+      booking: settings.db.url + '/' + settings.db.bookings
```

Comment:
Masz literówkę dlatego nie pobiera danych. Powinno być `settings.db.booking`. Porównaj z tym, co masz w obiekcie `db` w pliku `settings.js`
```suggestion
      booking: settings.db.url + '/' + settings.db.booking
```